test/multiverse/suites/rails/rails3_app/my_app.rb

```diff
@@ -94,6 +94,8 @@ class MyApp < Rails::Application
 
     post '/parameter_capture', :to => 'parameter_capture#create'
 
+    get '/view_components', :to => 'view_component#index'
```

Comment:
Maybe add a comment here about what this route is doing? I could see someone thinking it wasn't used and accidentally deleting it.

Reply:
Added! https://github.com/newrelic/newrelic-ruby-agent/pull/2367/commits/9fa2fb0d30b2ed490f571515083ba0da09a0cbe3